src/fancy/Grid.hx

```diff
@@ -147,10 +147,23 @@ class Grid {
     this.rows = rows;
```

Comment:
we could probably call `setRowsAndColumns` from the constructor instead of repeating everything twice.